### ISSUE TITLE
Keeps downloading first page over and over

### ISSUE THREAD

**Issue by brendandelany** [adrianlois/RedShelf-Bypass-PDFDownload#3]
It’s possible I’ve made a mistake, since I’m new to Python (and programming in general), but when I run the file in Terminal it just repeatedly downloads the first page while changing the filename. I’ve attached screenshots of the download folder for reference.
<img width="1032" alt="download folder screenshot #1" src="https://github.com/adrianlois/RedShelf-Bypass-DRM-PDFDownload/assets/132954548/0580845f-89a6-4740-ad2d-6ec1fc8200d1">
<img width="1032" alt="download folder screenshot #2" src="https://github.com/adrianlois/RedShelf-Bypass-DRM-PDFDownload/assets/132954548/489804ce-cdd2-4865-9fae-81c0ba35e175">
<img width="1032" alt="download folder screenshot #3" src="https://github.com/adrianlois/RedShelf-Bypass-DRM-PDFDownload/assets/132954548/7ef05b7f-eb72-4c2d-9ddc-fdb7cea212b0">
I can share the modified Python and JSON files if that helps. I can also screenshot my Terminal window and share that too. Just let me know.

**Comment by chropic** [comment]
Getting this same error.

**Comment by angelito-dev** [comment]
It is an error on your file, you must follow the instructions. They are ok. 

base_url = "https://platform.virdocs.com/rscontent/epub/XXXXXXX/XXXXXXX/OEBPS/images/page-{}.jpg"
pdf_file = "EXPORT_PDF_FILE.pdf"
numpag = 350


Modify ONLY the XXXX part and the "numpag" and it should work

**Comment by adrianlois** [comment]
Correct, the variable "numpag" indicates the last page of the document with reference to the URL. It should not be confused with the last page of the document itself. 

For example, if a book has 300 pages, it is possible that the last page referenced in the URL is a higher number that does not exactly match the page of the book (.../page-306.jpg") with 306 being the 300th page corresponding to the book.

**Comment by angelito-dev** [comment]
To clarify, this method did not work for all documents in RedShelf, some of them are not images but text and cannot be dowloaded. thank you very much adrianlois for the work done.
